README.md

```diff
@@ -443,6 +443,7 @@ These are the following services and checks perform by each Auditor. There are c
 | AWS_IAM_Auditor.py                     | IAM User                      | Do users have managed policies attached                                                |
 | AWS_IAM_Auditor.py                     | Password policy (Account)     | Does the IAM password policy meet or exceed<br>AWS CIS Foundations Benchmark standards |
 | AWS_IAM_Auditor.py                     | Server certs (Account)        | Are they any Server certificates stored by IAM                                         |
+| AWS_Lambda_Auditor.py                  | Lambda Function               |Has function been used or updated in the last 30 days                           |
```

Comment:
Minor style tweak... space between pipe and word in third column.  Has no effect on markdown render